manual/cops_style.md

```diff
@@ -6028,6 +6028,28 @@ WordRegex | (?-mix:\A[\p{Word}\n\t]+\z)
 
 * [https://github.com/bbatsov/ruby-style-guide#percent-w](https://github.com/bbatsov/ruby-style-guide#percent-w)
 
+## Style/YodaCondition
+
+Enabled by default | Supports autocorrection
+--- | ---
+Enabled | No
```

Comment:
Would it be that difficult to support auto-correction?  (I've never done it before, so I'm not implying it's easy.  Just asking...)